client/src/components/Creator.js

```diff
@@ -1,8 +1,150 @@
 import React, { Component } from "react";
+import { Container, Row, Col, Button, Card, Table } from "react-bootstrap";
+import { ImCopy, ImTwitter, ImTelegram } from "react-icons/im";
+import { TwitterShareButton, TelegramShareButton } from "react-share";
+import "./Creator.css";
+
+function getBalance() {
+  return "0";
+}
 
 class Creator extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getValue(weiVal) {
+    const val = (weiVal / Math.pow(10, 18)).toFixed(3);
```

Comment:
optional: We could also use `Web3.utils.fromWei(weiVal)` which is a bit more readable if you'd like